index.html

```diff
@@ -150,88 +155,105 @@ <h1 class="head-recent">My Recent <br> Works</h1>
             </div>
         </div>
     </section>
-    <img class="section-bg01" src="./Images//section-bg01.svg" alt="">
-    <img class="section-bg02" src="./Images//section-bg02.svg" alt="">
+    <img class="section-bg01" src="./images/section-bg01.svg" alt="">
     <section class="about">
         <div class="about-myself">
-            <div>
-                <h2>About me</h2>
-                <p>
-                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                    don’t hestiate to contact me.
-                </p>
-                <button class="btn-resume">Get My Resume</button>
-            </div>
-            
-            <div class="about-catalog">
-                <div class="languages"><img src="./Images//Icon-Langguage.svg" alt=""></div>
-                <p>Languages</p>
-                <ul>
-                    <li>Javascript</li>
-                    <li>Ruby</li>
-                    <li>Html</li>
-                    <li>CSS</li>
-                </ul>
+            <div class="about-me-card">
+                <div class="about-img-left">
+                    <img src="./images/illustration-about-me-2-desktop.svg" alt="">
+                </div>
+                <div class="about-me">
+                    <h2>About me</h2>
+                    <p>
+                        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                    </p>
+                    <button class="btn-resume">Get My Resume</button>
+                </div>
```

Comment:
- [ ] Your website looks wonderful. Well done 👏 
But there is a little issue that I see with the website's responsiveness. The goal of this project is to make your project very responsive across all views, from mobile to tablet and various laptop screens. I recorded a video that explains my observations. Kindly fix this issue. Please [watch the video here.](https://drive.google.com/file/d/1LbTyyf5aZWPAenEyQQKtvEqJNgrclvTN/view)

- Essentially, your website's about section is not centrally aligned in views between ``768px - 1075px`. This means that your visitor that uses tablets and the likes will not have a good experience with you site. Please ensure your site looks good across all screen types.